docs/utilities.html

```diff
@@ -321,6 +321,25 @@ <h5 id="media-print-shortcuts">
 <!--       -->
 <!--       --><span class="fake-comment">// This element would only appear when printed out</span>
 <!--       --><span>&lt;div class="visible-print"&gt;&lt;/div&gt;</span>
+<!--     --></pre>
+            <h5 id="accessibility-shortcuts">
+              <a href="#accessibility-shortcuts">
+                Accessibility Shortcuts
+              </a>
+            </h5>
+            <p>
+              Use these pre-made rules, to create elements that only screenreader users can see. 
+            </p>
+            <pre class="fake-code bottom-margin">
+<!--       --><span class="fake-comment">// .sr-only and .sr-only-focusable</span>
+<!--       -->
+<!--       --><span class="fake-comment">// Visually hidden element which is still accessible to users with a screenreader.</span>
+<!--       --><span class="fake-comment">// This can be used to give context to elements, such as a "Read more" button.</span>
+<!--       -->&lt;p class="sr-only"&gt;Hello world&lt;/p&gt;
```

Comment:
I quite liked your examples you listed in the PR text. I really don't have much experience with the Accessibility world and while this above example is understandable to me, it didn't inspire the actual use-case "a ha" that your other example did.

Could you replace this example with one similar to the one you listed in the PR?